server/a_socials/settings/dev.py

```diff
@@ -36,3 +36,5 @@
 
 # SET WEBPACK_LOADER_CACHE
 WEBPACK_LOADER['DEFAULT']['CACHE'] = not DEBUG
+
+ENVIRONMENT = "development"
```

Comment:
Well done @OsayamenOsaretin but this additional line of code is not really required, since we can use `dotenv.get('NODE_ENV')` from our .env file. Please check line 62 in /server/api/views.py file 
to see an existing implementation.

Reply:
@OsayamenOsaretin has this been verified?

Reply:
@OsayamenOsaretin how long has this been open?

Reply:
@andela-sjames, @tosmak16  Sorry for the late reply, I think making this explicit makes it clearer. Also, using the node environment doesn't seem to be ideal.
Finally, I think the flexibility of defining this is in the settings file allows it to be more flexible.